scripts/drs_file_access_examination.ipynb

```diff
@@ -0,0 +1,326 @@
+    "    DRS_url_information['drs_unresolved_error'] = []\n",
+    "    \n",
+    "    # # Creating the file_access_error in the DRS url information dictionary\n",
+    "    DRS_url_information['file_access_error'] = []\n",
+    "    \n",
+    "    # Calling Martha to resolved the DRS url\n",
+    "    martha_request = requests.post(\"https://us-central1-broad-dsde-prod.cloudfunctions.net/martha_v2\",\n",
+    "                                    data = {'url': DRS_url},\n",
+    "                                    headers={\"authorization\": \"Bearer \" + access_token[0]} )\n",
+    "\n",
+    "    # Getting the response from martha in json format\n",
+    "    martha_response = json.loads(martha_request.text)\n",
+    "    \n",
+    "    # Assigning the file path to None (It will be overwritten if a file path is found)\n",
+    "    DRS_url_information['file_path'] = None\n",
+    "    \n",
+    "    # Checking if the resolving failed\n",
+    "    if martha_request.status_code != 200:\n",
+    "        # Saving the error if resolving failed\n",
+    "        error_json = json.loads(martha_response['response']['text'])\n",
+    "        error = '\"' + error_json['error']['message'] + '\"'\n",
+    "        DRS_url_information['is_resolved'] = False\n",
+    "        DRS_url_information['drs_unresolved_error'].append(error)\n",
+    "        unresolved_DRS_urls_errors_list.append(error)\n",
+    "    else:\n",
+    "        # Saving the google bucket file path if resolving passed\n",
+    "        DRS_url_information['is_resolved'] = True\n",
+    "        # Get the file path that start with 'gs://'\n",
+    "        for values in martha_response['dos']['data_object']['urls']:\n",
+    "            if 'gs://' in values['url']:\n",
+    "                DRS_url_information['file_path'] = values['url']\n",
+    "    \n",
+    "    # Getting the first 30 bytes of the file in the goolge bucket\n",
+    "    try:\n",
+    "        drs.head(DRS_url, num_bytes=30)\n",
+    "        access_to_file = True\n",
+    "        files_with_access = files_with_access + 1\n",
+    "    except:\n",
+    "        # Saving the error if failed\n",
+    "        error = '\"' + str((sys.exc_info()[1])).split(\"Error:\")[1] + '\"'\n",
+    "        DRS_url_information['file_access_error'].append(error)\n",
+    "        files_without_access_errors_list.append(error)\n",
+    "        access_to_file = False\n",
+    "        files_without_access = files_without_access + 1\n",
+    "    \n",
+    "    # Checking if we got access to the file\n",
+    "    DRS_url_information['file_access'] = access_to_file\n",
+    "    \n",
+    "    # Increasing the resolved DRS urls count\n",
+    "    resolved_DRS_urls_count = resolved_DRS_urls_count + 1 \n",
+    "    \n",
+    "    # Outputing the resolved DRS urls count after it tried to resolved\n",
+    "    print(\"\")\n",
+    "    print(str(resolved_DRS_urls_count)+\"/\"+str(len(DRS_urls_dict)) + \" completed\")\n",
```

Comment:
small thing but adding spaces around the operators would make it clearer to read